src/index.html

```diff
@@ -7,8 +7,64 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="navigation-logo">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a href="#" class="navigation-link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a href="#" class="navigation-link">
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
        <ul class="navigation-list">
          <li class="navigation-item">
            <a href="#" class="navigation-link is-active">
              Apple
            </a>
          </li>
          
          <li class="navigation-item">
            <a href="#" class="navigation-link">
              Samsung
            </a>
          </li>
```
add spaces between sibling multiline blocks